src/include/miopen/solver.hpp

```diff
@@ -531,6 +531,11 @@ struct PerformanceImplicitGemmV4R4Fwd : Serializable<PerformanceImplicitGemmV4R4
     std::string ToString() const;
 };
 
+struct PerformanceImplicitGemmMlirCppFwd : PerformanceImplicitGemmV4R4Fwd
```

Comment:
Is it possible to begin WITHOUT tuning? We are adding too much new functionality in a single PR, which is error-prone and delays reviewing. Thanks.

Reply:
Hmmm, I'd love to update the PR if it helps the reviewer in any way. In this case, what does it mean to remove the tuning implementation of the solver? Does it mean I need to do the following:
 - Remove `search` and `GetPerformanceConfig` from the solver implementation
 - Remove this `PerformanceImplicitGemmMlirCppFwd`

If I do remove the above, then tuning on the child class will fall back to the parent class. This will make the code incorrect as tuning result on parent class will be written instead (of child class). Please give specific instructions on how to make everything else to behave correctly but without tuning. Thanks!

Reply:
(For the long term development goal) Let me start a email thread that copy you, @whchung and @asroy. We might be able to reuse the performance config of original solver instead of inheriting a new one.

(For the scope of this PR) When I had the discussion with @asroy  way backwards. I got some very strong opinion towards reusing existing solver or performance configs, in case things grow different in the future. It'd apply the same here I guess.

Reply:
Tuning removed